jsettlers.main.android/src/main/java/jsettlers/main/android/core/controls/AndroidControls.java

```diff
@@ -142,6 +142,11 @@ public String getDescriptionFor(UIPoint position) {
 
 	@Override
 	public void setMapViewport(MapRectangle screenArea) {
+		ShortPoint2D displayCenter = new ShortPoint2D(screenArea.getLineStartX(screenArea.getLines() / 2)
```

Comment:
We use a line length of 200 characters in the project. Can you adapt this line accordingly?